### PR TITLE
feat(coc): add v-mode selection format

### PR DESCRIPTION
**Pull request by wroyca**
## 📑 Description

Add `<leader>l;` to format selected lines in visual mode.

## Additional Information

I believe `<leader>;` would be more natural here, but I opted to follow the mnemonic as it was already established

### CODE REVIEW

File: lua/astrocommunity/lsp/coc-nvim/init.lua [hunk 1]
```diff
@@ -40,6 +40,7 @@ return {
         maps.n["<Leader>lR"] = maps.n.gr
         maps.n["<Leader>lr"] = { "<Plug>(coc-rename)", desc = "Rename current symbol" }
         maps.n["<Leader>lf"] = { function() vim.cmd.Format() end, desc = "Format buffer", nowait = true }
+        maps.x["<Leader>l;"] = { "<Plug>(coc-format-selected)", desc = "Format selection", nowait = true }
```
Comment (mehalter) [inline lua/astrocommunity/lsp/coc-nvim/init.lua:43]:
```suggestion
        maps.x["<Leader>lF"] = { "<Plug>(coc-format-selected)", desc = "Format selection", nowait = true }
```

Maybe this makes more sense?

File: lua/astrocommunity/lsp/coc-nvim/init.lua [hunk 2]
```diff
@@ -40,6 +40,7 @@ return {
         maps.n["<Leader>lR"] = maps.n.gr
         maps.n["<Leader>lr"] = { "<Plug>(coc-rename)", desc = "Rename current symbol" }
         maps.n["<Leader>lf"] = { function() vim.cmd.Format() end, desc = "Format buffer", nowait = true }
```
Comment (mehalter) [inline lua/astrocommunity/lsp/coc-nvim/init.lua:42]:
```suggestion
        maps.n["<Leader>lf"] = { function() vim.cmd.Format() end, desc = "Format buffer" }
```

hehe this is an accident, maybe fix this too in this PR :')

**Review by mehalter — CHANGES_REQUESTED** [review]

**Comment by wroyca** [comment]
Will squash

**Review by mehalter — APPROVED** [review]
LGTM

**Pull request merged**